Makefile

```diff
@@ -292,6 +325,10 @@ remote-watcher:
 	set -e; . ./bin/variables && \
 	_build/dev/rel/watcher/bin/watcher remote_console
 
+remote-watcher-informational:
+	set -e; . ./bin/variables && \
+	_build/dev/rel/watcher/bin/watcher remote_console
```

Comment:
```suggestion
	_build/dev/rel/watcher/bin/watcher_informational remote_console
```

?

Reply:
Hmm. Not that easy! Needs to match with whats in rel/config.exs

Reply:
Meh. They can't be started separately anyway (for now).